src/gtk-3.20/scss/apps/_nemo.scss

```diff
@@ -17,6 +17,10 @@
     }
 
     .nemo-window {
+        .nemo-inactive-pane .view { 
+            background-color: $dark_bg_color; 
```

Comment:
shouldn't you also set `color: $dark-fg_color;`?

Reply:
or, instead, use:

```scss
            background-color: mix($base_color, $text_color, .12);
```

Reply:
Thanks - sorry I didn't consider the color - I generally have the same for $fg_color and $dark_fg_color

The mix option is better though...